docs/api/ReactWrapper/reduce.md

```diff
@@ -41,8 +41,8 @@ class Foo extends React.Component {
 
 ```jsx
 const wrapper = mount(<Foo />);
-const total = wrapper.find(Bar).reduce((amount, n) => amount + n.prop('amount'));
-expect(total).to.equal(16);
+const total = wrapper.find(Bar).reduce((amount, n) => amount ? amount + n.prop('amount') : n.prop('amount'));
```

Comment:
in other words, `[1, 2, 3].reduce(function (prev, item) { console.log([prev, item]); });` logs `[1, 2]` and then `[undefined, 3]`